source/nodejs/adaptivecards-designer/src/containers/viva-connections/viva-connections-container.ts

```diff
@@ -16,23 +26,26 @@ abstract class VivaConnectionsContainer extends HostContainer {
         hostElement.appendChild(vcContainer);
     }
 
+    public getHostConfig(): Adaptive.HostConfig {
```

Comment:
Possibly a clean solution is to add a new class in the middle which implements getHost, getBackgroundColor, supportsMultipleThemes, ... and default constructor.
```
class ThemeSupportedHostContainer extends HostContainer
{
     constructor(darkBackground, lightBackground);
}
class VivaConnectionsContainer extends ThemeSupportedHostContainer{
  super(color1, color2, name, somethingelse)
}
```